src/main/resources/META-INF/aop.xml

```diff
@@ -3,10 +3,17 @@
 <aspectj>
   <aspects>
     <aspect name="kamon.jdbc.instrumentation.StatementInstrumentation"/>
+    <aspect name="kamon.jdbc.instrumentation.HikariInstrumentation"/>
   </aspects>
 
-    <weaver>
-       <include within="java.sql.Statement..*"/>
-       <include within="java.sql.Connection..*"/>
-   </weaver>
+
+  <weaver>
+    <include within="com.zaxxer.hikari.pool.HikariPool"/>
+    <include within="com.zaxxer.hikari.pool.ProxyConnection"/>
+
+    <!-- JDBC Drivers -->
+    <include within="org.h2..*"/>
+    <include within="org.postgresql.jdbc4..*"/>
+    <include within="com.microsoft.sqlserver.jdbc..*"/>
```

Comment:
I think is a good idea also include the `MariaDB(org.mariadb.jdbc..)` and `MySQL(com.mysql.jdbc..)` drivers. @ivantopo WDYT?